akka-remote/src/main/scala/akka/remote/artery/ArteryTransport.scala

```diff
@@ -419,7 +444,7 @@ private[remote] class ArteryTransport(_system: ExtendedActorSystem, _provider: R
     else {
       associations.computeIfAbsent(remoteAddress, new JFunction[Address, Association] {
         override def apply(remoteAddress: Address): Association = {
-          val newAssociation = new Association(ArteryTransport.this, materializer, remoteAddress, controlSubject)
+          val newAssociation = new Association(ArteryTransport.this, materializer, remoteAddress, controlSubject, largeMessageDestinationsEnabled, largeMessageDestinations)
```

Comment:
I think it can be alright to recompute `largeMessageDestinationsEnabled` in `Association`